js/plates-view/plate-label.js

```diff
@@ -0,0 +1,62 @@
+import * as THREE from 'three'
+import { hueAndElevationToRgb } from '../colormaps'
+import config from '../config'
+
+const RADIUS = 1.025
+
+function pointTexture (label, color) {
```

Comment:
The biggest change I was considering is refactoring this label to avoid overlap with the cross section points

Reply:
Blind guess, but if you play with the radius, it might be an easy tweak. Otherwise, we'd need separate rendering layer which is quite painful. I don't think this project is using that yet.

Reply:
Oh right sorry, I meant trying to refactor the duplicated code :)